test/skate_web/controllers/page_controller_test.exs

```diff
@@ -173,7 +173,7 @@ defmodule SkateWeb.PageControllerTest do
       conn = get(conn, "/")
 
       assert html_response(conn, 200) =~
-               "data-map-limits=\"{&quot;east&quot;:3,&quot;north&quot;:1,&quot;south&quot;:2,&quot;west&quot;:4}\""
+               "data-map-limits=\"{&quot;north&quot;:1,&quot;south&quot;:2,&quot;east&quot;:3,&quot;west&quot;:4}\""
```

Comment:
A possible option could be asserting on the substrings that are guaranteed to be there:
```
assert html_response(conn, 200) =~ "data-map-limits=\"{&quot;"
assert html_response(conn, 200) =~ "&quot;east&quot;:3"
assert html_response(conn, 200) =~ "&quot;north&quot;:1"
assert html_response(conn, 200) =~ "&quot;south&quot;:2"
assert html_response(conn, 200) =~ "&quot;west&quot;:4"
```

This has the downside that it's _technically_ not as strong an assertion as what we currently have, but also what we currently have is a little too strong, since I don't think there are any guarantees that the keys will stay in that order.

But also happy to wait on changing this until this test actually starts flaking (if it ever does).